.github/fixtures/test-from-context/expected.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [unreleased]
+
+### Test
+
+- Add tests
+
+## [0.2.0] - 2022-04-06 - This is so awesome!
+
+### Bug Fixes
+
+- Fix feature 2 (that was a tough one)
+
+### Features
+
+- Add feature 2
+
+## [0.1.0] - 2022-04-06
```

Comment:
I keep getting 2022-04-05 in the tests - probably because of some timezone issues. Locally, I get the same behavior in `test-commit-preprocessors` as well. What should I do?

Reply:
Ah, it's probably because the `context.json` was created in a different timezone. I'll update this.